WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -149,7 +149,8 @@
       </li>
       <li>
         <a href="https://live.worldcubeassociation.org/" target="_blank" class="top-nav hide-new-window-icon">
-          <%= ui_icon('sort numeric down') %> WCA Live
+           <%= ui_icon('sort numeric down') %> 
```

Comment:
I think the trailing space at the end of this line is causing the tests to fail.

Reply:
Fixed, tests have passed.